### PR TITLE
Nextjs prefixed routing

### PR DESCRIPTION
**Pull request by rjkerrison**
### NextJs Prefixing routing

[Airtable ticket](https://airtable.com/tbl7CISHKkr8Kdeh2/viwlDIU6cp8cbQYhj/recMy4v4Im1vLFwPN)

Resolves issues around prefixed feature environments

route

### Acceptance Criteria

- [x] Links between pages work on a branch environment
- [ ] Search result loading routing navigation works on a branch environment
- [ ] Pagination routing works on a branch environment

### Testing information

<insert notes for testers that might be helpful>

### Pre-flight Checklist

- [ ] Tests
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/web/next.config.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 require('dotenv').config();
 
+const isMasterEnvironment =
```
Comment (m-doughty) [inline medicines/web/next.config.js]: I'm really not a fan of code being aware of CI/CD stuff, can we pass in BASE_URL from the Azure build process and keep the code agnostic to implementation details?

Comment (rjkerrison) [reply medicines/web/next.config.js]: that wouldn't take care of the need for `/index.html` for links to static pages to work in branch environments, so we'd have to bring that in too

**Comment by craiga** [comment]
@rjkerrison This is well out of date now. I suspect routing has moved on. Time to close it?

**Comment by rjkerrison** [comment]
Well out-of-date. Closing.

**Pull request closed without merging**